packages/financial-templates-lib/src/clients/InsuredBridgeL1Client.ts

```diff
@@ -72,21 +74,29 @@ export class InsuredBridgeL1Client {
       .flat();
   }
 
-  getRelayedDepositsForL1Token(l1Token: string) {
+  getRelayedDepositsForL1Token(l1Token: string): Relay[] {
     this._throwIfNotInitialized();
     return Object.values(this.relays[l1Token]);
   }
 
-  getPendingRelayedDeposits() {
+  getPendingRelayedDeposits(): Relay[] {
     return this.getAllRelayedDeposits().filter((relay: Relay) => relay.relayState === relayState.Pending);
   }
 
-  getPendingRelayedDepositsForL1Token(l1Token: string) {
+  getPendingRelayedDepositsForL1Token(l1Token: string): Relay[] {
     return this.getRelayedDepositsForL1Token(l1Token).filter((relay: Relay) => relay.relayState === relayState.Pending);
   }
 
   // TODO: we might want to add other accessors that do other forms of filtering.
 
+  calculateRealizedLpFeesPctForDeposit(/* deposit: any*/): string {
+    return this.toWei("0.05");
```

Comment:
good placeholder for now, this function might get chunky

Reply:
ye, for sure. we still need to decide exactly how we want to structure this as well. I think it'll likely follow as closely to compounds model as posible. see https://compound.finance/documents/Compound.Whitepaper.pdf for the section on `Interest Rate Model`.